js/microprofile-config-callback.js

```diff
@@ -260,7 +374,8 @@ var microprofileConfigCallBack = (function() {
                 stepName = stepContent.getCurrentStepName();
             }
             var editorFileName;
-            if(stepName === "EnableMPConfig") {
+            if((stepName === "EnableMPConfig") || (stepName === "ConfigureAsVariableElement" ||
```

Comment:
if ((stepName === "EnableMPConfig") || (stepName === "ConfigureAsVariableElement") ||
                (stepName === "ConfigureAsappPropElement")) { ?

Reply:
Removed the extraneous parenthesis.